src/py/flwr/cli/new/templates/app/code/client.hf.py.tpl

```diff
@@ -0,0 +1,55 @@
+"""$project_name: A Flower / HuggingFace Transformers app."""
+
+import flwr as fl
+from transformers import AutoModelForSequenceClassification
+
+from $project_name.task import (
+    get_weights,
+    load_data,
+    set_weights,
+    train,
+    test,
+    CHECKPOINT,
+    DEVICE,
+)
+
+
+# Flower client
+class FlowerClient(fl.client.NumPyClient):
+    def __init__(self, net, trainloader, testloader):
+        self.net = net
+        self.trainloader = trainloader
+        self.testloader = testloader
+
+    def get_parameters(self, config):
+        return get_weights(self.net)
+
+    def set_parameters(self, parameters):
+        set_weights(self.net, parameters)
+
+    def fit(self, parameters, config):
+        self.set_parameters(parameters)
+        train(self.net, self.trainloader, epochs=1)
+        return self.get_parameters(config={}), len(self.trainloader), {}
+
+    def evaluate(self, parameters, config):
+        self.set_parameters(parameters)
+        loss, accuracy = test(self.net, self.testloader)
+        return float(loss), len(self.testloader), {"accuracy": accuracy}
+
+
+def client_fn(cid):
+    # Load model and data
+    net = AutoModelForSequenceClassification.from_pretrained(
+        CHECKPOINT, num_labels=2
+    ).to(DEVICE)
+    trainloader, valloader = load_data(int(cid), 2)
```

Comment:
How does the `cid` is passed here? I think in the original example we had `argparse`?

Reply:
Ok, got it. It'd be good to have that information somewhere, but since it relates to all simulation examples, maybe there's one common place that we can add it? 
Maybe it'd also be worth having the information that the example can't be run "manually" = launching each file yourself.